TweakAccessorGenerator/TweakAccessorGenerator/Core/TweakAccessorCodeGenerator.swift

```diff
@@ -117,139 +116,20 @@ extension TweakAccessorCodeGenerator {
-                                                                   customTweakProvidersSetupCode: customTweakProvidersSetupCode)
-        
-        if configuration.usePropertyWrappers {
-            return """
-                static let tweakManager: TweakManager = {
-            \(tweakProvidersCodeBlock)
-                    let tweakManager = TweakManager(tweakProviders: tweakProviders)
-                    tweakManager.useCache = \(configuration.shouldCacheTweaks)
-                    return tweakManager
-                }()
-
-                var tweakManager: TweakManager {
-                    return Self.tweakManager
-                }
-            """
-        }
-        else {
-            return """
-                lazy var tweakManager: TweakManager = {
-            \(tweakProvidersCodeBlock)
-                    let tweakManager = TweakManager(tweakProviders: tweakProviders)
-                    tweakManager.useCache = \(configuration.shouldCacheTweaks)
-                    return tweakManager
-                }()
-            """
-        }
-    }
-    
-    private func tweakProvidersCodeBlock(with configuration: Configuration,
-                                         customTweakProvidersSetupCode: [Filename: CodeBlock]) -> String {
-        let grouping = Dictionary(grouping: configuration.tweakProviders) { $0.type }
-        
-        var tweakProvidersString: [String] = [
-            """
-                    var tweakProviders: [TweakProvider] = []\n
-            """
-        ]
-        
-        var currentIndexByConf: [String: Int] = grouping.mapValues{ _ in 0 }
-        
-        for tweakProvider in configuration.tweakProviders {
-            let value = grouping[tweakProvider.type]!
-            let index = currentIndexByConf[tweakProvider.type]!
-            let tweakProvider = value[index]
-            let tweakProviderName = "\(tweakProvider.type.lowercasedFirstChar())_\(index+1)"
-            var generatedString: [String] = []
-            let macros = tweakProvider.macros?.joined(separator: " || ")
-            
-            let jsonFileURL = "jsonFileURL_\(index+1)"
-            let headerComment = """
-                        // \(tweakProvider.type)
-                """
-            generatedString.append(headerComment)
-                
-            if macros != nil {
-                let macroStarting = """
-                        #if \(macros!)
-                """
-                generatedString.append(macroStarting)
-            }
-            
-            switch tweakProvider.type {
-            case "EphemeralTweakProvider":
-                let tweakProviderAllocation =
-                    """
-                            let \(tweakProviderName) = NSMutableDictionary()
-                            tweakProviders.append(\(tweakProviderName))
-                    """
-                generatedString.append(tweakProviderAllocation)
+    private func tweakManagerCodeBlock() -> String {
```

Comment:
nit: same here could use var here if you fancied

Reply:
The rest of the file has functions, feels better to keep them all as functions IMHO. The fact that there are no parameters here is less important than having overall consistency in this file I would say.